File: WEB/web-app/src/models/DTO_DetalleCuentaJSON.ts
import { DTO_Fila_Detalle } from "./DTO_Fila_Detalle";
import { DTO_Param } from "./DTO_Param";

export class DTO_DetalleCuentaJSON {
    filas: DTO_Fila_Detalle[] = [];
    descuento: DTO_Param = {nombre: 'Monto', valor: ''} as DTO_Param;
    impuesto: DTO_Param =  {nombre: 'Porcentaje', valor: ''} as DTO_Param;
}

File: APP/DTO/DTO_DetalleCuentaJSON.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO
{
    public class DTO_DetalleCuentaJSON
    {
        #region Properties
        public List<DTO_Fila_Detalle> Filas { get; set; } = new();
        public DTO_Param Descuento { get; set; } = new();
        public DTO_Param Impuesto { get; set; } = new();
        #endregion


    }
}
